### PR TITLE
[READY] Mob Proc Cleanup

### PR DESCRIPTION
**Pull request by uraniummeltdown**
Seems to work fine

Moved many mob procs to their own defense files, made them more OOP, moved is helpers to the is helpers file, added in missing attack anims and logging
Made human/attack_hand species based and split intents into their own procs, so different species can have different effects when grabbed, harmed, disarmed, etc. An example is Wryn who can have their hive node ripped out if cuffed and harm intented. CPR moved to its own proc
Removed some unnecessary alien and brain radiation definitions, they play off carbon now
Slime emotes log, slimes can steal nutrition if they attack other slimes, slimes can change their mood/face with new emotes, fixed a bug with mousedrop
You can use reagents plasma and epinephrine to control slime mutation chance
Using docility potion no longer makes a simple animal slime, but makes the target docile and non hungry

:cl:
add: Slimes can be injected with plasma reagent and epinephrine to control mutation chance. Plasma increases mutation chance 5% per 5u to a max of 50%, epinephrine decreases by 5% per 5u to a minimum of 0%
add: Slimes can change their face using new emotes
add: Slimes can steal nutrition from other slimes when attacking them
add: Slime docility potion no longer makes a simple animal slime, instead makes the target docile and never hungry
/:cl:

### CODE REVIEW

**Comment by Birdtalon** [comment]
You're on fire 🔥

**Comment by uraniummeltdown** [comment]
@Fox-McCloud fixed merge conflicts

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 1]
```diff
@@ -315,8 +315,172 @@
...
 	return
 
-/datum/species/proc/handle_attack_hand(var/mob/living/carbon/human/H, var/mob/living/carbon/human/M) //Handles any species-specific attackhand events.
-	return
+/datum/species/proc/help(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+	if(attacker_style && attacker_style.help_act(user, target))//adminfu only...
+		return 1
+	if(target.health >= config.health_threshold_crit)
+		target.help_shake_act(user)
+		add_attack_logs(user, target, "Shaked")
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/species/species.dm]: Please put this in `carbon/help_shake_act` and change it to `add_attack_logs(M, src, "Shaked", admin_notify = FALSE)`

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 2]
```diff
@@ -315,8 +315,172 @@
...
+		add_attack_logs(user, target, "vampirebit")
+		return
+		//end vampire codes
+	if(attacker_style && attacker_style.harm_act(user, target))
+		return 1
+	else
+		var/datum/unarmed_attack/attack = user.species.unarmed
+
+		user.do_attack_animation(target)
+		add_attack_logs(user, target, "Melee attacked with fists")
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/species/species.dm]: Add `, admin_notify = target.ckey ? TRUE : FALSE)` to the end of this

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 3]
```diff
@@ -315,8 +315,172 @@
...
+			target.apply_effect(4, WEAKEN, armor_block)
+			target.forcesay(hit_appends)
+		else if(target.lying)
+			target.forcesay(hit_appends)
+
+/datum/species/proc/disarm(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+	if(attacker_style && attacker_style.disarm_act(user, target))
+		return 1
+	else
+		add_attack_logs(user, target, "Disarmed")
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/species/species.dm]: Add `admin_notify = FALSE` to the end of this

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 4]
```diff
@@ -315,8 +315,172 @@
...
+		user.do_attack_animation(target)
+		if(target.w_uniform)
+			target.w_uniform.add_fingerprint(user)
+		var/obj/item/organ/external/affecting = target.get_organ(ran_zone(user.zone_sel.selecting))
+		var/randn = rand(1, 100)
+		if(randn <= 25)
+			target.apply_effect(2, WEAKEN, target.run_armor_check(affecting, "melee"))
+			playsound(target.loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+			target.visible_message("<span class='danger'>[user] has pushed [target]!</span>")
+			add_attack_logs(user, target, "Pushed over")
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/species/species.dm]: Add `admin_notify = FALSE` to the end of this

File: code/modules/mob/living/living_defense.dm [hunk 5]
```diff
@@ -229,13 +229,13 @@
...
 	if(user == src || anchored)
 		return 0
 	if(!(status_flags & CANPUSH))
 		return 0
 
-	for(var/obj/item/grab/G in src.grabbed_by)
+	for(var/obj/item/grab/G in grabbed_by)
 		if(G.assailant == user)
 			to_chat(user, "<span class='notice'>You already grabbed [src].</span>")
 			return
```
Comment (tigercat2000) [inline code/modules/mob/living/living_defense.dm]: The `add_attack_logs` directly below this, please add `admin_notify = FALSE` to

**Pull request merged**